src/integration-test/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectRevisionControllerIntegrationTest.java

```diff
@@ -240,6 +242,8 @@ public void testGetWorkflowRevisionShouldReturnSavedWorkflowRevision() {
         System.out.println(responseString);
 
         System.out.println(secondCatalogObjectRevisionCommitTime.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME));
+        String encodedObjectName = URLEncoder.encode(secondCatalogObjectRevision.get("name").toString(), "UTF-8")
+                                             .replace("+", "%20");
```

Comment:
How comes "+" is the only character to replace?

Reply:
Yes. It's a tricky part. I explained why we need this replacement.
"'+' is accepted by RequestParam as space. In case of PathVariable, '+' is accepted as '+'"

And the URLEncoder implements the HTML Specifications for how to encode URLs in HTML forms
https://stackoverflow.com/questions/4737841/urlencoder-not-able-to-translate-space-character